notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>The last bullet point is a bit vague. Is this specific to the spectrum? So, for example, in the spectrum chosen you bring attention to a possible emission line. Could this be something like "Identify spectral emission lines" - or is the 4th bullet point necessary at all? I'd also specify for the 3rd bullet point exactly what kind of obsevation we are using, because the instructions may be specific. "Learn to plot a Hubble spectrum" maybe?</p><p> </p><p>For the 2nd one - should it be called a Historical Observational Coverage Plot here? That clearly defines what will be made.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>